### PR TITLE
Add MLH Externships

### PR DESCRIPTION
**Pull request by alphaX86**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Use this repo to share and keep track of any tech-related internships. For a [Go
...
-| [MLH](https://fellowship.mlh.io/) | Various Remote | [Explorer](https://fellowship.mlh.io/programs/explorer) and [Open Source](https://fellowship.mlh.io/programs/open-source) Fellowships, Fall, Spring, Summer available & [MLH Coach](https://mlh.io/coaches) positons now available (optional if you want to help others)|
+| [MLH](https://fellowship.mlh.io/) | Various Remote | [Explorer](https://fellowship.mlh.io/programs/explorer) and [Open Source](https://fellowship.mlh.io/programs/open-source) Fellowships, Fall, Spring, Summer available (UPDATE: Externships now available for Fall, Spring and Summer! Refer [MLH](https://mlh.io) site for more info & [MLH Coach](https://mlh.io/coaches) positons now available (optional if you want to help others)|
```
Comment (alphaX86) [inline README.md:89]: Externships now available. Info posted live

Comment (Zmwang622) [reply README.md:89]: do you have a more specific url?

Comment (alphaX86) [reply README.md]: Wait a sec.....

File: README.md [hunk 2]
```diff
@@ -86,7 +86,7 @@ Use this repo to share and keep track of any tech-related internships. For a [Go
...
-| [MLH](https://fellowship.mlh.io/) | Various Remote | [Explorer](https://fellowship.mlh.io/programs/explorer) and [Open Source](https://fellowship.mlh.io/programs/open-source) Fellowships, Fall, Spring, Summer available & [MLH Coach](https://mlh.io/coaches) positons now available (optional if you want to help others)|
+| [MLH](https://fellowship.mlh.io/) | Various Remote | [Explorer](https://fellowship.mlh.io/programs/explorer) and [Open Source](https://fellowship.mlh.io/programs/open-source) Fellowships, Fall, Spring, Summer available (UPDATE: Externships now available for Fall, Spring and Summer! Refer [MLH externships](https://fellowship.mlh.io/programs/externship) site for more info) & [MLH Coach](https://mlh.io/coaches) positons now available (optional if you want to help others)|
```
Comment (alphaX86) [inline README.md:89]: Link added

File: README.md (hunk 1 above)
Comment (alphaX86) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Zmwang622) [reply README.md:89]: awesome, thanks!

**Pull request merged**